backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VirtualQueueController.java

```diff
@@ -36,14 +33,14 @@
 public class VirtualQueueController {
     @Autowired
     public VirtualQueueService virtualQueueService;
-    
+
     @PostMapping("/shop")
     public CreateShopResponse addShop(@RequestBody CreateShopRequest createShopRequest) {
         return virtualQueueService.createShop(createShopRequest);
     }
 
-    @GetMapping("/token/{shopId}")
```

Comment:
Can you ensure that this change has the PR merged by Ishita: https://github.com/googleinterns/gpay-virtual-queue/pull/19/files